Fix build with modern autoconf/automake and fix a bashism in configure.
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,3 +1,5 @@
+ACLOCAL_AMFLAGS = -I m4
+
 SUBDIRS = m4 docs lash liblash lashd clients icons pylash
 
 pkgconfigdir = $(libdir)/pkgconfig
--- a/configure.ac
+++ b/configure.ac
@@ -2,7 +2,6 @@ AC_INIT([LASH],[0.5.4])
 AC_CONFIG_SRCDIR([lash/types.h])
 AC_CONFIG_HEADER([config.h])
 AM_INIT_AUTOMAKE
-AM_ACLOCAL_INCLUDE([m4])
 
 ### Check for programs ###
 AC_LANG([C])
@@ -161,10 +160,10 @@ AC_ARG_ENABLE(pylash, [AS_HELP_STRING(--disable-pylash, [Force disable pylash bu
 if test x$build_pylash != xdisabled; then
   AM_PATH_PYTHON(2.3, python_found=yes, python_found=no)
 
-  if test x$python_found == xyes; then
+  if test x$python_found = xyes; then
     AM_CHECK_PYTHON_HEADERS(, python_found=no)
 
-    if test x$python_found == xyes; then
+    if test x$python_found = xyes; then
       AC_PROG_SWIG(1.3.31)
       if test "$SWIG_LIB" ; then
         build_pylash=yes
